### DISCUSSION TITLE
VoiScribe: iOS App for On-Device Speech Recognition

### DISCUSSION THREAD

**Discussion by yasutak** [openai/whisper#1265] [Show and tell]
We are delighted to introduce VoiScribe, an iOS application for on-device speech recognition. Built upon the powerful whisper.cpp, VoiScribe brings secure and efficient speech transcription directly to your iPhone or iPad.

Highlighted features of VoiScribe include:
* Secure offline speech recognition using Whisper
* Intuitive and refined user interface for seamless navigation
* Effortless transcription editing capabilities
* Compatible with iPhones and iPads (Mac version under development)
* Export options for both audio files and transcriptions
* Upcoming iCloud synchronization

As we prepare for VoiScribe's public release, we are currently seeking beta testers to provide valuable feedback. If you are interested in joining our beta testing program, please follow this link: https://testflight.apple.com/join/LhaENwM5 to install the beta version of the app.

Your support is greatly appreciated, and we eagerly anticipate your thoughts on VoiScribe. Thank you in advance for your time and consideration!

<p align="middle">
<img src="https://user-images.githubusercontent.com/19920981/233530454-4bbf67f1-95b5-4aff-b819-ab0e19d77260.jpeg" width=49% height=49%>

<img src="https://user-images.githubusercontent.com/19920981/233530462-83599d34-1877-4092-83bf-9e5a46c0db83.jpeg" width=49% height=49%>
</p>

<img src="https://user-images.githubusercontent.com/19920981/233529782-27b99ba6-164e-4bac-bd07-c684dbcfefd5.png" width=50% height=50%>

**Comment by maks554** [comment]
а как без пригласительного кода скачать это приложение? скиньте его
